tests/desktop/test_details_page.py

```diff
@@ -21,76 +19,76 @@ class TestDetails:
     @pytest.mark.nondestructive
     def test_that_register_login_link_is_present_in_addon_details_page(self, mozwebqa):
         details_page = Details(mozwebqa, "Firebug")
-        Assert.true(details_page.header.is_register_link_visible, "Register link is not visible")
-        Assert.true(details_page.header.is_login_link_visible, "Login links is not visible")
+        assert details_page.header.is_register_link_visible, "Register link is not visible"
+        assert details_page.header.is_login_link_visible, "Login links is not visible"
 
     @pytest.mark.action_chains
     @pytest.mark.nondestructive
     def test_that_dropdown_menu_is_present_after_click_on_other_apps(self, mozwebqa):
         details_page = Details(mozwebqa, "Firebug")
-        Assert.equal(details_page.header.menu_name, "Other Applications")
+        assert details_page.header.menu_name, "Other Applications"
         details_page.header.hover_over_other_apps_menu()
-        Assert.true(details_page.header.is_other_apps_dropdown_menu_visible)
+        assert details_page.header.is_other_apps_dropdown_menu_visible
 
     @pytest.mark.nondestructive
     def test_that_addon_name_is_displayed(self, mozwebqa):
         details_page = Details(mozwebqa, "Firebug")
         # check that the name is not empty
-        Assert.not_none(details_page.title, "")
+        assert details_page.title, "" is not None
```

Comment:
This should be: `assert details_page.title is not None` - ignore the weird empty message argument.